### PR TITLE
Twt shell cmd doc update

### PR DESCRIPTION
**Pull request by ajayparida**

### CODE REVIEW

File: doc/nrf/working_with_nrf/nrf70/developing/powersave.rst [hunk 1]
```diff
@@ -403,8 +403,8 @@ The following shell commands and network management APIs are provided for power
    * - net_mgmt(NET_REQUEST_WIFI_TWT)
      - wifi twt setup 0 0 1 1 0 1 1 1 65000 52400
      - | Set up TWT:
-       | TWT wake interval - 524000 µs
-       | TWT wake duration - 65000 µs
+       | TWT wake interval - 65000 µs
+       | TWT interval - 52400 µs
```
Comment (krish2718) [inline doc/nrf/working_with_nrf/nrf70/developing/powersave.rst:407]: should be `524000`

File: samples/wifi/shell/README.rst [hunk 2]
```diff
@@ -71,24 +71,25 @@ Supported CLI commands
...
-     - Get the statistics of the Wi-Fi interface
-   * - ap_enable
-     - Configure the Wi-Fi interface as access point mode
-   * - ap_disable
-     - Configure the Wi-Fi interface as station mode
+     - Wi-Fi interface statistics
+   * - ap
+     - | Access Point mode commands
+       | enable - Configure the Wi-Fi interface as access point mode
+       | disable - Configure the Wi-Fi interface as station mode
```
Comment (sachinthegreen) [inline samples/wifi/shell/README.rst:91]: Match description to the one from the shell

File: samples/wifi/shell/README.rst [hunk 3]
```diff
@@ -71,24 +71,25 @@ Supported CLI commands
...
    * - statistics
-     - Get the statistics of the Wi-Fi interface
-   * - ap_enable
-     - Configure the Wi-Fi interface as access point mode
-   * - ap_disable
-     - Configure the Wi-Fi interface as station mode
+     - Wi-Fi interface statistics
+   * - ap
+     - | Access Point mode commands
+       | enable - Configure the Wi-Fi interface as access point mode
```
Comment (sachinthegreen) [inline samples/wifi/shell/README.rst:90]: Enable Access Point mode, with the following parameters: 
<List parameters from the shell>

File: samples/wifi/shell/README.rst [hunk 4]
```diff
@@ -71,24 +71,25 @@ Supported CLI commands
...
+       | disable - Configure the Wi-Fi interface as station mode
+       | Currently access point mode is not supported
```
Comment (sachinthegreen) [inline samples/wifi/shell/README.rst:92]: This should be a note, i.e. Note : Access point mode is presently unsupported.

**Review by sachinthegreen — CHANGES_REQUESTED** [review]

**Review by sachinthegreen — APPROVED** [review]

**Review by krish2718 — APPROVED** [review]

File: samples/wifi/shell/README.rst [hunk 5]
```diff
@@ -71,24 +71,29 @@ Supported CLI commands
...
+     - Wi-Fi interface statistics
+   * - ap
+     - | Access Point mode commands
+       | enable - Enable Access Point mode with below parameters
+       | <SSID>
+       | <SSID length>
+       | channel>
+       | <psk> [optional]
+       | disable - Disable Access Point mode
+       | (Note: Access point mode is presently not supported.)
```
Comment (richabp) [inline samples/wifi/shell/README.rst:96]:
```suggestion
       | (Note that the Access Point mode is presently not supported.)
```

File: samples/wifi/shell/README.rst [hunk 6]
```diff
@@ -71,24 +71,29 @@ Supported CLI commands
...
    * - statistics
-     - Get the statistics of the Wi-Fi interface
-   * - ap_enable
-     - Configure the Wi-Fi interface as access point mode
-   * - ap_disable
-     - Configure the Wi-Fi interface as station mode
+     - Wi-Fi interface statistics
+   * - ap
+     - | Access Point mode commands
+       | enable - Enable Access Point mode with below parameters
```
Comment (richabp) [inline samples/wifi/shell/README.rst:90]:
```suggestion
       | enable - Enable Access Point mode, with the following parameters:
```

File: samples/wifi/shell/README.rst [hunk 7]
```diff
@@ -71,24 +71,29 @@ Supported CLI commands
    * - Subcommands
      - Description
    * - scan
-     - Scan for access points in the vicinity
+     - Scan for Wi-Fi APs
    * - connect
-     - | Connect to an access point with below parameters:
+     - | Connect to Wi-Fi AP with below parameters:
```
Comment (richabp) [inline samples/wifi/shell/README.rst:76]:
```suggestion
     - | Connect to the Wi-Fi AP with the following parameters:
```

File: samples/wifi/shell/README.rst [hunk 8]
```diff
@@ -104,13 +109,14 @@ Supported CLI commands
        | setup - Start a TWT flow:
        | <negotiation_type: 0 - Individual, 1 - Broadcast, 2 - Wake TBTT>
        | <setup_cmd: 0 - Request, 1 - Suggest, 2 - Demand>
-       | <dialog_token> <flow_id> <responder> <trigger> <implicit>
-       | <announce> <twt_wake_interval> <twt_interval>
+       | <dialog_token: 1-255> <flow_id: 0-7> <responder: 0/1> <trigger: 0/1>
+       | <implicit: 0/1> <announce: 0/1> <twt_wake_interval: 1µs-262144µs>
+       | <twt_interval: 1µs-2^31µs>
```
Comment (richabp) [inline samples/wifi/shell/README.rst:114]:
```suggestion
       | <implicit: 0/1> <announce: 0/1> <twt_wake_interval: 1 µs to 262144 µs>
       | <twt_interval: 1 µs to 2^31 µs>
```

File: samples/wifi/shell/README.rst [hunk 9]
```diff
@@ -104,13 +109,14 @@ Supported CLI commands
...
        |
        | teardown - Teardown a TWT flow:
        | <negotiation_type: 0 - Individual, 1 - Broadcast, 2 - Wake TBTT>
        | <setup_cmd: 0 - Request, 1 - Suggest, 2 - Demand>
-       | <dialog_token> <flow_id>
+       | <dialog_token: 1-255> <flow_id: 0-7>
```
Comment (richabp) [inline samples/wifi/shell/README.rst:119]:
```suggestion
       | <dialog_token: 1 - 255> <flow_id: 0 - 7>
```

File: samples/wifi/shell/README.rst [hunk 10]
```diff
@@ -104,13 +109,14 @@ Supported CLI commands
        | setup - Start a TWT flow:
        | <negotiation_type: 0 - Individual, 1 - Broadcast, 2 - Wake TBTT>
        | <setup_cmd: 0 - Request, 1 - Suggest, 2 - Demand>
-       | <dialog_token> <flow_id> <responder> <trigger> <implicit>
-       | <announce> <twt_wake_interval> <twt_interval>
+       | <dialog_token: 1-255> <flow_id: 0-7> <responder: 0/1> <trigger: 0/1>
```
Comment (richabp) [inline samples/wifi/shell/README.rst:112]:
```suggestion
       | <dialog_token: 1 - 255> <flow_id: 0 - 7> <responder: 0/1> <trigger: 0/1>
```

**Review by richabp — CHANGES_REQUESTED** [review]

**Comment by krish2718** [comment]
@richabp  the help is taken from the source code, so, if you have comments then it needs to be fixed in upstream and then cherry-picked, so, for now this has to exactly match the help command and we can fix the help (and update everywhere) later.

**Comment by richabp** [comment]
> @richabp the help is taken from the source code, so, if you have comments then it needs to be fixed in upstream and then cherry-picked, so, for now this has to exactly match the help command and we can fix the help (and update everywhere) later.
> 
> FYI its here https://github.com/zephyrproject-rtos/zephyr/blob/main/subsys/net/l2/wifi/wifi_shell.c#L1059

Get it, thanks for clarifying.

**Review by richabp — APPROVED** [review]

**Review by kittydepa — APPROVED** [review]

**Comment by carlescufi** [comment]
@ajayparida do you want to rebase this?

**Pull request merged**